### PR TITLE
Website Tweaks

### PR DESCRIPTION
**Pull request by meltahawy**
— Changed the “Chaos” text color to red (more suiting).
— Added a notification telling the user to hover over the logo, it is
hard to tell it is animated otherwise.
— Added a moving background that adds more to the “chaos”. (based on
Canva Moving Background from Codepen and it is a background I also use
for my own website.)

### CODE REVIEW

**Comment by o3LL** [comment]
Any screenshot ?
Can you refactor without using jQuery ?

**Comment by meltahawy** [comment]
Yup, here's a screenshot:

screen shot 2017-05-26 at 6 07 23 pm

It looks much better when the particles in the background are actually moving.

I have no idea how to refactor without jQuery, sorry. But I don't mind anyone taking my code and doing that.

**Comment by hongaar** [comment]
> It looks much better when the particles in the background are actually moving.

Convince us with a gif!

File: server/index.html [hunk 1]
```diff
@@ -57,6 +69,7 @@ <h1 class="title"><strong>Hello, Chaos!</strong></h1>
                     </div>
                     <canvas id="canvas"></canvas>
                     <p class="subtitle">Hi, this is <strong>ChaosBot</strong>.</p>
+                    <p class="content">^^^ Hover over me.</p>
```
Comment (hongaar) [inline server/index.html]: Not really chaotic, think visitors should figure this out on their own

File: server/static/css/main.css [hunk 2]
```diff
@@ -58,4 +71,25 @@ body.transform {
   transform:rotate(180deg);
   -ms-transform:rotate(180deg);
   -webkit-transform:rotate(180deg);
-}
\ No newline at end of file
+}
+
+/* BACKGROUND */
```
Comment (hongaar) [inline server/static/css/main.css:69]: Can you refactor this into a separate `background.css` file?

File: server/index.html [hunk 3]
```diff
@@ -42,12 +42,24 @@
...
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
 </head>
 <body>
     <a href="https://github.com/chaosbot/chaos">
         <img id="github-ribbon" src="https://camo.githubusercontent.com/652c5b9acfaddf3a9c326fa6bde407b87f7be0f4/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6f72616e67655f6666373630302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png">
     </a>
+
     <div class="container">
+      <div id="bg">
```
Comment (hongaar) [inline server/index.html]: Maybe use a single `<canvas id="background"></canvas>` instead. Would be much cleaner.

File: server/static/css/main.css (hunk 2 above)
Comment (o3LL) [reply server/static/css/main.css:69]: Bad idea, the main.css file is still small, we dont need one more http request just for the background. I think keeping background css code in main.css is great

**Comment by o3LL** [comment]
> I have no idea how to refactor without jQuery, sorry. But I don't mind anyone taking my code and doing that.  

I'm doing it then !  ;) (if pr is accepted)

**Comment by meltahawy** [comment]
Here's a GIF:
http://gph.to/2qWkqie
chaos bot website

File: server/index.html (hunk 1 above)
Comment (meltahawy) [reply server/index.html]: I had no idea there was an animation until I realized there was a JS file. I do agree with you though, there is probably a neater way to alert visitors. :)

File: server/static/css/main.css (hunk 2 above)
Comment (hongaar) [reply server/static/css/main.css:69]: Oh! Fine. Let's use SASS/LESS at some point...

Comment (o3LL) [reply server/static/css/main.css:69]: Why not

**Comment by o3LL** [comment]
@meltahawy can you add me as Collaborator on your fork ?

**Comment by meltahawy** [comment]
@MUCHZER Added you!

**Comment by o3LL** [comment]
Here you go, refactor script without jquery and made some style update.

**Pull request merged**